examples/window/window_settings.rs

```diff
@@ -37,6 +40,12 @@ fn main() {
         .run();
 }
 
+/// Add an icon to the window task bar.  Only works in Windows and Linux.
+fn setup(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let icon_handle = asset_server.load("branding/icon.png");
+    commands.spawn(WindowIcon(Some(icon_handle)));
+}
```

Comment:
It might be possible to simplify all of this by adding an `icon: Option<PathBuf>` or something fancier to the `Window` struct and then adding these systems when an existing bevy plugin runs.  But first I thought it would be good to check whether I got the general idea right.

Reply:
Yeah, let's keep this simple here.

Reply:
> Yeah, let's keep this simple here.

By "simple," are you thinking API-simple (adding a field to `Window`), or implementation-simple (sticking with the current approach)?

Reply:
Yeah, I meant that I prefer the current approach :)

Reply:
Once it occurred to me that several moving parts would need to be orchestrated in order to make this work without issues, I took the liberty of making a plugin.  I'm not attached to it, though, so no problem if it's dropped.